node/pkg/p2p/p2p.go

```diff
@@ -351,6 +364,9 @@ func Run(
 					gst.Cleanup()
 				case <-ctx.Done():
 					return
+				case ccqErr := <-ccqErrC:
```

Comment:
just curious, why does the err channel handler go here? it to make this stop on error?

Reply:
Fixed.